src/crimson/osd/osd.cc

```diff
@@ -1036,6 +1046,15 @@ seastar::future<> OSD::handle_rep_op_reply(crimson::net::Connection* conn,
   return seastar::now();
 }
 
+seastar::future<> OSD::handle_mark_me_down(crimson::net::Connection* conn,
```

Comment:
why do we need to handle `MOSDMarkMeDown` on the OSD side? i think this message is supposed to be handled by monitor.